.github/workflows/regression-test.yml

```
@@ -0,0 +1,19 @@
+---
+name: Regression tests
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+
+jobs:
+  regression-tests:
+    uses: glass-dev/glass-benchmarks/.github/workflows/regression-test.yaml@v0
+    with:
+      starting_revision: ${{github.event.pull_request.base.ref}}
+      end_revision: ${{github.event.pull_request.head.ref}}
```

These are normally formatted like this

```suggestion
      starting_revision: ${{ github.event.pull_request.base.ref }}
      end_revision: ${{ github.event.pull_request.head.ref }}
```